avro-serializer/src/test/java/io/confluent/kafka/serializers/KafkaAvroSerializerTest.java

```diff
@@ -232,4 +238,41 @@ public void testInvalidInput() {
       // this is expected
     }
   }
+
+  @Test
+  public void testEnableAutoSchemaRegistrationWhenSchemaVersionExists() {
+    IndexedRecord avroRecord = createAvroRecord();
+    byte [] bytes = null;
+
+    bytes = avroSerializer.serialize(topic, avroRecord);
+
+    assertNotNull(bytes);
+    Object obj = avroDecoder.fromBytes(bytes);
+    assertEquals(avroRecord, obj);
+  }
+
+  @Test
+  public void testDisableAutoSchemaRegistrationWhenNoVersion() {
```

Comment:
This seems to be the same as testEnableAutoSchemaRegistrationWhenSchemaVersionExists above